src/PrestaShopBundle/Resources/views/Admin/Component/Layout/employee_dropdown.html.twig

```diff
@@ -0,0 +1,73 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+{% if employee is defined %}
+  <div class="dropdown employee-dropdown">
+    <div class="rounded-circle person" data-toggle="dropdown">
+      <i class="material-icons">account_circle</i>
+    </div>
+    <div class="dropdown-menu dropdown-menu-right">
+      <div class="employee-wrapper-avatar">
+        <div class="employee-top">
+          <span class="employee-avatar">
+            <img class="avatar rounded-circle" src="{{ employee.getImage }}" alt="{{ employee.firstname }}" />
+          </span>
+          <span class="employee_profile">
+            {{ 'Welcome back %name%'|trans({'%name%': employee.firstname}, 'Admin.Navigation.Header') }}
+          </span>
+        </div>
+
+        <a class="dropdown-item employee-link profile-link" href="{{ link.getAdminLink('AdminEmployees', true, {}, {'id_employee' : employee.id, 'updateemployee' : 1})|escape('html_attr') }}">
```

Comment:
Actually, this one is migrated so we can use the correct route, or if you prefer it can wait for phase 2
```suggestion
        <a class="dropdown-item employee-link profile-link" href="{{ path('admin_employees_edit', {employeeId: employee.id}) }}">
```

Reply:
Same. I let you decide

Reply:
Done